okhttp-android-support/src/main/java/com/squareup/okhttp/internal/huc/JavaApiConverter.java

```diff
@@ -727,7 +718,11 @@ public Object getContent(Class[] classes) throws IOException {
 
     @Override
     public InputStream getInputStream() throws IOException {
-      throw throwResponseBodyAccessException();
+      return new InputStream() {
+        @Override public int read() throws IOException {
+          throw throwResponseBodyAccessException();
```

Comment:
I'm most unsure about the changes in this file because the larger semantics of this are unclear to me. As you can see, this is the _only_ usage which necessitated the `throws IOException` declaration. Nothing in OkHttp core needs it.